angelspetrochem/prototypes/buildings/air-filter.lua

```diff
@@ -117,13 +198,13 @@ data:extend({
     },
     allowed_effects = { "consumption", "speed", "productivity", "pollution" },
     crafting_categories = { "petrochem-air-filtering" },
-    crafting_speed = 2,
+    crafting_speed = 2.25,
```

Comment:
I already mentioned to not make air filters faster
```suggestion
    crafting_speed = 2,
```